attentive-android-sdk/src/main/java/com/attentive/androidsdk/creatives/Creative.java

```diff
@@ -87,6 +85,42 @@ public void trigger() {
         webView.loadUrl(url);
     }
 
+    private String buildCompanyCreativeUrl() {
+        Uri.Builder uriBuilder =
+            getCompanyCreativeUriBuilder(attentiveConfig.getDomain(), attentiveConfig.getMode());
+
+        UserIdentifiers userIdentifiers = attentiveConfig.getUserIdentifiers();
+
+        addUserIdentifiersAsParameters(uriBuilder, userIdentifiers);
+
+        return uriBuilder.build().toString();
+    }
+
+    private void addUserIdentifiersAsParameters(Uri.Builder builder,
+                                                UserIdentifiers userIdentifiers) {
+        if (userIdentifiers.getVisitorId() != null) {
+            builder.appendQueryParameter("visitor_id", userIdentifiers.getVisitorId());
+        } else {
+            Log.e(this.getClass().getName(), "No VisitorId found. This should not happen.");
+        }
+
+        if (userIdentifiers.getClientUserId() != null) {
+            builder.appendQueryParameter("client_user_id", userIdentifiers.getClientUserId());
+        }
+        if (userIdentifiers.getPhone() != null) {
+            builder.appendQueryParameter("phone", userIdentifiers.getPhone());
+        }
+        if (userIdentifiers.getEmail() != null) {
+            builder.appendQueryParameter("email", userIdentifiers.getEmail());
+        }
+        if (userIdentifiers.getKlaviyoId() != null) {
+            builder.appendQueryParameter("klaviyo_id", userIdentifiers.getKlaviyoId());
+        }
+        if (userIdentifiers.getShopifyId() != null) {
+            builder.appendQueryParameter("shopify_id", userIdentifiers.getShopifyId());
+        }
```

Comment:
i think we're missing the custom identifiers here?

Reply:
😶‍🌫️ yes, you're right. thank you! going to send it as a JSON array

Reply:
Updated again to just send a JSON dictionary